File: frontend/src/views/Antecedentes/seccionesDetalles/DetallesDatosVivienda.js
import { Typography } from '@material-ui/core';
import React from 'react'

export default function DetallesDatosVivienda (props)
{
    const classes = props.classes;
    const detalle = props.detalle;
    console.log(detalle.serviciosBasicos);

    return (
        <div id="datosVivienda">
            <Typography variant="h5"  className={classes.subtitulo}>Vivienda</Typography>
            <Typography variant="body1" className={detalle.casa ? classes.normal : classes.faltante}>
                <strong className={classes.normal}>Casa: </strong>
                {detalle.casa ? detalle.casa : "No registrado"}
            </Typography>
            <Typography variant="body1" className={detalle.serviciosBasicos != null ? classes.normal : classes.faltante}>
                <strong className={classes.normal}>Servicios Básicos: </strong>
                {detalle.serviciosBasicos !== null ? detalle.serviciosBasicos === 1 ? "Sí" : "No" : "No registrado"}
            </Typography>
        </div>
    );
}

File: movil/lib/screens/Beneficiaries/Antecedentes/Secciones/DetallesDatosVivienda.dart
import 'package:flutter/material.dart';
import 'package:movil/classes/Antecedentes.dart';

class DetallesDatosVivienda extends StatelessWidget {
  final Antecedentes antecedentes;
  DetallesDatosVivienda({Key key, @required this.antecedentes}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Column(
      children: <Widget>[
        Padding(
          padding: EdgeInsets.symmetric(vertical: 5.0),
          child: Text(
            "Datos de Vivienda",
            style: TextStyle(
              fontSize: 18,
              decoration: TextDecoration.underline,
            ),
          ),
        ),
        //Casa
        Row(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Text(
              "Casa: ",
              style: TextStyle(
                fontSize: 16,
                fontWeight: FontWeight.bold,
              ),
            ),
            Text(
              antecedentes.casa != null ? antecedentes.casa : 'No registrado',
              style: TextStyle(
                fontSize: 16,
                fontStyle: antecedentes.casa != null ? FontStyle.normal : FontStyle.italic,
              ),
            ),
          ],
        ),
        //Servicios Básicos
        Row(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Text(
              "Servicios Básicos: ",
              style: TextStyle(
                fontSize: 16,
                fontWeight: FontWeight.bold,
              ),
            ),
            Text(
              antecedentes.serviciosBasicos != null ? (antecedentes.serviciosBasicos == 1 ? "Sí" : "No") : 'No registrado',
              style: TextStyle(
                fontSize: 16,
                fontStyle: antecedentes.serviciosBasicos != null ? FontStyle.normal : FontStyle.italic,
              ),
            ),
          ],
        ),
      ],
    );
  }
}